packages/model-runtime/src/utils/streams/openai/openai.ts

```diff
@@ -57,8 +57,9 @@ const transformOpenAIStream = (
 
     const errorData = {
       body: chunk,
-      type: 'errorType' in chunk ? chunk.errorType : AgentRuntimeErrorType.ProviderBizError,
-    } as ChatMessageError;
+      message: 'message' in chunk ? typeof chunk.message === 'string' ? chunk.message : JSON.stringify(chunk) : JSON.stringify(chunk),
+      type: 'errorType' in chunk ? chunk.errorType as ChatMessageError['type'] : AgentRuntimeErrorType.ProviderBizError,
```

Comment:
if chunk has errorType, it should be the `ProviderBizError` as it's provider specific error.

and the message in message is lgtm